File: top-k-frequent-elements/top-k-frequent-elements.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        cnt  = []
        num = []
        res = []
        for i in nums:
            if i not in num:
                num.append(i)
                res.append(nums.count(i))
        for j in range(k):
            m =max(res)
            j = res.index(m)
            cnt.append(num[j])
            res[j] = -1
        return cnt
        

File: 347-top-k-frequent-elements/top-k-frequent-elements.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        vector<int>res ; 
        unordered_map<int,int>mp ;
        for(int i : nums){
            mp[i]++;
        }
         priority_queue<pair<int,int>> p;
         for(auto it : mp){
             p.push({it.second,it.first});
         }
         while(k--){
             res.push_back(p.top().second);
             p.pop();
         }
         return res;
    }
};